.github/workflows/prettier-check.yml

```diff
@@ -0,0 +1,31 @@
+name: autofix.ci
+on:
+  push:
+    branches:
+      - trunk
+    paths:
+      - "**.html"
+      - "**.css"
+      - "**.js"
+      - "**.md"
+      - "**.tsx"
+      - "**.json"
```

Comment:
these apply as a logical AND
https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#running-your-workflow-only-when-a-push-to-specific-branches-occurs

Reply:
Push action is restricted much more heavily now, as I noticed that it was actively being evaluated on all branches in the background, which is undesirable, and would use our monthly allotment of open-source free actions more quickly than is necessary.